src/common/components/repository-row/index.js

```diff
@@ -296,7 +296,7 @@ class RepositoryRow extends Component {
         actionSpinner = true;
         actionText = 'Checking...';
       } else {
-        actionText = 'Register';
+        actionText = 'Register Name';
```

Comment:
Does "Name" needs to be in title case? I believe we are not using title case anywhere?